.github/workflows/on-prerelease.yml

```diff
@@ -98,6 +98,7 @@ jobs:
     needs: pack
     permissions:
       actions: read
+      contents: read
```

Comment:
Should this be `contents: write`?

Reply:
It was contents: write before, but It was throwing the error, I'm seeing if contents read will make any difference.

Reply:
Rollback to contents:write